opentelemetry-api/src/opentelemetry/context/__init__.py

```diff
@@ -145,7 +145,9 @@ def __repr__(self):
 __all__ = ['Context']
 
 
-Context: typing.Optional[BaseRuntimeContext]
+Context = (  # pylint: disable=invalid-name
```

Comment:
This makes me wonder if we shouldn't relax the pylint rule for "constants"  (maybe even to `.*`) since there are so many things it demands UPPER_CASE for which are not really constants. Then this could be a single line without the awkward parens.

Reply:
+1

Reply:
It's done. Please review b05f347f3d08dee342b9ec239477123fa4e3a69f.

Reply:
Related PR: #108